packages/esm-form-entry-app/src/app/app.module.ts

```diff
@@ -31,6 +32,7 @@ import { FormCreationService } from './form-creation/form-creation.service';
     MonthlyScheduleResourceService,
     ConfigResourceService,
     SingleSpaPropsService,
+    TranslateService,
```

Comment:
I think we might need some additional `imports` for the injection to work.

Definitely use `HttpClientModule` here.

Also Angular is at 12, but the ngx translate is at 13. Not sure if they aligned, but sometimes that is a hint..

Reply:
I have added more imports and the error is now gone, thanks @FlorianRappl